_extensions/pkg/pkg.lua

```diff
@@ -0,0 +1,92 @@
+# generated with pharmaversedown::write_pkg_info()
+pkg_info = {}
```

Comment:
This is generated with `pharmaversedown::write_pkg_info()` that grabs data from the r-universe api: https://pharmaverse.r-universe.dev/api

`write_pkg_info()` could live in this repo somewhere if that's easier, but I believe this blog should have a companion package to host a few functions, instead of duplicating `appendix.R` in all posts ...

Reply:
Does not have to be `pharmaversedown`